### PR TITLE
docs(chore): update ZH translations of some bridge/connector fields (rv5.0)

### PR DESCRIPTION
**Pull request by thalesmg**
## Note: targetting `release-50`

Fixes https://emqx.atlassian.net/browse/EMQX-9368

See also:
https://emqx.atlassian.net/wiki/spaces/EDOC/pages/566268175/EMQX+Dashboard+UI+Improvement

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` and `.zh.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf [hunk 1]
```diff
@@ -636,10 +639,11 @@ emqx_ee_bridge_kafka {
                 "<code>none</code> Uses the key from the Kafka message unchanged."
                 "  Note: in this case, then the key must be a valid UTF-8 string.\n"
                 "<code>base64</code> Uses base-64 encoding on the received key."
-            zh: "定义了在通过MQTT转发之前如何处理Kafka消息的 Key。"
-                "<code>none</code> 使用Kafka消息中的 Key 原始值，不进行编码。"
-                "  注意：在这种情况下，Key 必须是一个有效的UTF-8字符串。\n"
-                "<code>base64</code> 对收到的密钥或值使用base-64编码。"
+            zh: "桥接到 EMQX 之前如何处理 Kafka 消息的键或值。\n"
```
Comment (thalesmg) [inline lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf:642]: I think the product team used an old version of this doc when the key and value descriptions were the same.  Please help fix "key" and "value" usages here and below.  :pray:

File: lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf [hunk 2]
```diff
@@ -653,10 +657,11 @@ emqx_ee_bridge_kafka {
...
                 "<code>base64</code> Uses base-64 encoding on the received value."
-            zh: "定义了在通过MQTT转发之前如何处理Kafka消息的 Value。"
-                "<code>none</code> 使用Kafka消息中的 Value 原始值，不进行编码。"
-                "  注意：在这种情况下，Value 必须是一个有效的UTF-8字符串。\n"
-                "<code>base64</code> 对收到的 Value 使用base-64编码。"
+            zh: "桥接到 EMQX 之前如何处理 Kafka 消息的键或值。\n"
+                "- <code>none</code>：直接使用 Kafka 消息中的键或值，"
+                "不做任何修改。 注意：在这种情况下，键或值字段必须是有效的 "
+                "UTF-8 字符串。\n"
+                "- <code>base64</code>：对收到的键或值进行 Base64 编码。"
```
Comment (zmstone) [inline lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf:664]: the old is good.

File: lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf [hunk 3]
```diff
@@ -636,10 +639,11 @@ emqx_ee_bridge_kafka {
...
                 "<code>base64</code> Uses base-64 encoding on the received key."
-            zh: "定义了在通过MQTT转发之前如何处理Kafka消息的 Key。"
-                "<code>none</code> 使用Kafka消息中的 Key 原始值，不进行编码。"
-                "  注意：在这种情况下，Key 必须是一个有效的UTF-8字符串。\n"
-                "<code>base64</code> 对收到的密钥或值使用base-64编码。"
+            zh: "桥接到 EMQX 之前如何处理 Kafka 消息的键或值。\n"
+                "- <code>none</code>：直接使用 Kafka 消息中的键或值，"
+                "不做任何修改。 注意：在这种情况下，键或值字段必须是有效的 "
+                "UTF-8 字符串。\n"
+                "- <code>base64</code>：对收到的键或值进行 Base64 编码。"
```
Comment (zmstone) [inline lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf:646]: the old is good.

File: lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf [hunk 4]
```diff
@@ -590,7 +591,9 @@ emqx_ee_bridge_kafka {
     consumer_max_rejoin_attempts {
         desc {
             en: "Maximum number of times allowed for a member to re-join the group. If the consumer group can not reach balance after this configured number of attempts, the consumer group member will restart after a delay."
-            zh: "允许一个成员重新加入小组的最大次数。如果超过改配置次数后仍不能成功加入消费组，则会在延迟一段时间后再重试。"
+            zh: "分区重平衡时，尝试重新加入消费者的最大次数。"
+                "如果尝试次数超过了这个限制时消费组仍然无法平衡分配分区，"
+                "则该成员将在等待一段时间后再重试。"
```
Comment (zmstone) [inline lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf:596]:
```suggestion
            zh: "允许一个消费组成员重试加入小组的最大次数。如果超过改配置次数后仍不能成功加入消费组，则会在延迟一段时间后再重试。"
```

File: lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf [hunk 5]
```diff
@@ -326,9 +324,12 @@ emqx_ee_bridge_kafka {
             en: "Partition strategy is to tell the producer how to dispatch messages to Kafka partitions.\n\n"
                 "<code>random</code>: Randomly pick a partition for each message\n"
                 "<code>key_dispatch</code>: Hash Kafka message key to a partition number"
-            zh: "设置消息发布时应该如何选择 Kafka 分区。\n\n"
-                "<code>random</code>: 为每个消息随机选择一个分区。\n"
-                "<code>key_dispatch</code>: Hash Kafka message key to a partition number"
+            zh: "消息发送到分区策略。\n"
+                "- <code>random</code>：简单轮询策略，将消息均匀地发送到所有分区中，"
```
Comment (zmstone) [inline lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf:328]:
```suggestion
                "- <code>random</code>：将消息随机分发送到所有分区中，"
```

**Review by zmstone — COMMENTED** [review]
the old doc is maybe not perfect.
but the new ones are wrong.

radom translated to round_robin
consumer group translated to partitions
group rebalance translated to partition rebalance

**Pull request closed without merging**